### ISSUE TITLE
Error occurs in Android O !

### ISSUE THREAD

**Issue by HarborZeng** [apl-devs/AppIntro#449]
06-13 21:38:01.276 14459-14459/{my package name} E/AndroidRuntime: FATAL EXCEPTION: main
                                                                          Process:{my package name}, PID: 14459
                                                                          java.lang.ArrayIndexOutOfBoundsException: Array index out of range: 0
                                                                              at java.util.Vector.get(Vector.java:748)
                                                                              at com.github.paolorotolo.appintro.PagerAdapter.getItem(PagerAdapter.java:32)
                                                                              at com.github.paolorotolo.appintro.AppIntroBase$4.run(AppIntroBase.java:192)
                                                                              at android.os.Handler.handleCallback(Handler.java:751)
                                                                              at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                              at android.os.Looper.loop(Looper.java:154)
                                                                              at android.app.ActivityThread.main(ActivityThread.java:6236)
                                                                              at java.lang.reflect.Method.invoke(Native Method)
                                                                              at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:891)
                                                                              at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:781)

**this is my build.gradle file:**
```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 26
    buildToolsVersion "26.0.0"
    defaultConfig {
        applicationId "{my package name}"
        minSdkVersion 15
        targetSdkVersion 26
        versionCode 201706121
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    buildTypes {
        release {
            minifyEnabled true
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}
allprojects {
    repositories {
        maven { url 'https://jitpack.io' }
    }
}
dependencies {
    compile fileTree(include: ['*.jar'], dir: 'libs')
    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
        exclude group: 'com.android.support', module: 'support-annotations'
    })
    compile 'com.android.support:appcompat-v7:26.0.0-alpha1'
    compile 'com.android.support:design:26.0.0-alpha1'
    compile 'com.android.support.constraint:constraint-layout:1.0.2'
    testCompile 'junit:junit:4.12'
    compile 'com.jakewharton:butterknife:8.6.0'
    annotationProcessor 'com.jakewharton:butterknife-compiler:8.6.0'
    compile files('libs/qrcode.jar')
    compile 'com.github.apl-devs:appintro:v4.2.0'
}
```

**Comment by avluis** [comment]
AppIntro has not been tested on Android O yet so this should be expected.
I'm not seeing your code though.

**Comment by HarborZeng** [comment]
@

**Comment by avluis** [comment]
@HarborZeng What was the issue? Incompatible tools?

**Comment by HarborZeng** [comment]
@avluis 
well...not a native speaker, you might not understand what I was talking about...
Anywany, after I add ```compile 'com.github.apl-devs:appintro:v4.2.0'``` to my build.gradle, then I click *Sync Now* the exception: ArrayIndexOutOfBoundsException occurs. At that time I even didn't write any code using your library.

It might be a compatibility problem for using a Alpha version support library.

So, I lowered *compileSdkVersion* and *targetSdkVersion* to 25, and support library of course. Then the problen solved.

Then I found that people seems are less likely to .... you konw.
